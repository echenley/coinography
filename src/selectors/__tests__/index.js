import * as selectors from '../index'

describe('selectors', () => {
  it('exports correct selectors', () => {
    expect(selectors).toMatchSnapshot()
  })
})
