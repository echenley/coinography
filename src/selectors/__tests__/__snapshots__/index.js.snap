// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`selectors exports correct selectors 1`] = `
Object {
  "getCandleChartData": [Function],
  "getChartDataLoadingState": [Function],
  "getChartType": [Function],
  "getCurrency": [Function],
  "getGranularity": [Function],
  "getLineChartData": [Function],
  "getRawChartData": [Function],
}
`;
